File: WorkGuideBack/DAL/Entities/Lesson.cs
namespace DAL.Entities
{
    public class Lesson
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public int OrderNumber { get; set; }

        public Guid CourseId { get; set; }

        public Course Course { get; set; }

        public HashSet<Theory> TheoryPages { get; set; }

        public HashSet<Test> TestPages { get; set; }

        public HashSet<UserLessonScore> UsersLessonScores { get; set; }
    }
}

File: workguidefront/src/Types/Lesson.ts
import Test from "./Test";
import Theory from "./Theory";

type Lesson = {
  id: string;
  name: string;
  orderNumber: number;
  courseUrl: string;
  isComplexTest: boolean;
  courseId: string;
  courseName: string;
  theoryPages: Theory[];
  testPages: Test[];
  finished: boolean;
};

export default Lesson;